events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+            activeIcon: TABLET_BUTTON_PRESSED
+        });
+
+    var open = false,
+        intervalBoard,
+        questionText,
+        choiceTexts = [],
+        answerText,
+        triviaData,
+        request = Script.require('../entities/modules/request.js').request,
+        type = null,
+        category = null,
+        difficulty = null,
+        currentChoices = [],
+        lights = [],
+        correctHighlights = [],
+        timer,
+        gameZone,
+        gameZoneProperties,
+        avatarCounter,
+        bubble,
+        introPlayed = false,
+        correctCount = null,
+        prizeDisplay,
+        prizeMoney,
+        winnerID = null,
+        correctColor = null;
+
+    var htmlEnDeCode = (function() {
+        var charToEntityRegex,
+            entityToCharRegex,
+            charToEntity,
+            entityToChar;
+    
+        function resetCharacterEntities() {
+            charToEntity = {};
+            entityToChar = {};
+            // add the default set
+            addCharacterEntities({
+                '&amp;': '&',
+                '&gt;': '>',
+                '&lt;': '<',
+                '&quot;': '"',
+                '&#39;': "'",
+                '&lsquo;': '"',
+                '&rsquo;': '"',
+                '&ldquo;': '"',
+                '&rdquo;': '"',
+                '&eacute;': 'e',
+                '&prime;': '\'',
+                '&Prime': '"'
+            });
+        }
+    
+        function addCharacterEntities(newEntities) {
+            var charKeys = [],
+                entityKeys = [],
+                key, encodedChar;
+            for (key in newEntities) {
+                encodedChar = newEntities[key];
+                entityToChar[key] = encodedChar;
+                charToEntity[encodedChar] = key;
+                charKeys.push(encodedChar);
+                entityKeys.push(key);
+            }
+            charToEntityRegex = new RegExp('(' + charKeys.join('|') + ')', 'g');
+            entityToCharRegex = new RegExp('(' + entityKeys.join('|') + '|&#[0-9]{1,5};' + ')', 'g');
+        }
+    
+        function htmlEncode(value) {
+            var htmlEncodeReplace = function(match, capture) {
+                return charToEntity[capture];
+            };
+    
+            return (!value) ? value : String(value).replace(charToEntityRegex, htmlEncodeReplace);
+        }
+    
+        function htmlDecode(value) {
+            var htmlDecodeReplaceFn = function(match, capture) {
+                return (capture in entityToChar) ? entityToChar[capture] : String.fromCharCode(parseInt(capture.substr(2), 10));
+            };
+    
+            return (!value) ? value : String(value).replace(entityToCharRegex, htmlDecodeReplaceFn);
+        }
+    
+        resetCharacterEntities();
+    
+        return {
+            htmlEncode: htmlEncode,
+            htmlDecode: htmlDecode
+        };
+    })();
+
+    function begin() {
+        findTargets();
```

Comment:
what happens if this fails to find all the entities?

Reply:
Presumably it would mean some entities don't get changed by the script.  I can't recall this happening by such a failure though.

Reply:
i would like to see some error handling here, but i think it's OK to ignore for right now. perhaps put in the README some things that are TODOs?

Reply:
Yes, I will list items to do in future updates in the README.